plasma_framework/contracts/src/vaults/verifiers/Erc20DepositVerifier.sol

```diff
@@ -7,19 +7,19 @@ import {PaymentTransactionModel as DepositTx} from "../../transactions/PaymentTr
 import {PaymentOutputModel as DepositOutputModel} from "../../transactions/outputs/PaymentOutputModel.sol";
 
 /**
- * @notice implementation of Erc20 deposit verifier using Payment transaction as the deposit tx
+ * @notice Implementation of Erc20 deposit verifier using payment transaction as the deposit tx
  */
 contract Erc20DepositVerifier is IErc20DepositVerifier {
     using DepositOutputModel for DepositOutputModel.Output;
 
-    // hardcoded tx type for Payment Transaction
+    // Hardcoded transaction type for payment transaction
     uint8 constant private DEPOSIT_TX_TYPE = 1;
 
     /**
      * @notice Overrides the function of IErc20DepositVerifier and implements the verification logic
-     *         for Payment transaction
-     * @dev ERC20 token must be approved to the vault address beforehand
-     * @return verified (owner, token, amount) of the deposit ERC20 token data
+     *         for payment transaction
+     * @dev ERC20 token requires pre-approval to the vault address
```

Comment:
`Vault address must be approved to transfer from the depositor's address before doing the deposit` or similr